ndscan/dashboard/argument_editor.py

```diff
@@ -1134,10 +1239,8 @@ def write_to_params(self, params: dict) -> None:
                 "randomise_order": self.check_randomise.isChecked()
             }
         }
-        if (lower := schema["spec"].get("min", None)) is not None:
-            spec["range"]["limit_lower"] = lower
-        if (upper := schema["spec"].get("max", None)) is not None:
-            spec["range"]["limit_upper"] = upper
+        spec["range"]["limit_lower"] = self.min / self.scale
+        spec["range"]["limit_upper"] = self.max / self.scale
```

Comment:
Dividing by scale here looks wrong (this is following the ARTIQ convention where values are in base units and scale is just applied for display purposes).